frappe/core/doctype/doctype/doctype.js

```diff
@@ -135,6 +144,40 @@ frappe.ui.form.on("DocType", {
 			frm.form_wrapper.removeClass("mb-1");
 		}
 	},
+
+	async trim_table(frm) {
```

Comment:
Don't copy paste the code, move it to https://github.com/ankush/frappe/blob/db_migr/frappe/public/js/frappe/doctype/index.js